### PR TITLE
[DOCS-808] Adding first draft of guide

### PR DESCRIPTION
**Pull request by l0k0ms**
### What does this PR do?

Adds first draft of guide for synthetics downloaded test assertions.

### Motivation

PM request.

### Preview link

* https://docs-staging.datadoghq.com/gus/synthetics-guide/synthetics/guide/testing-a-downloaded-file/

### CODE REVIEW

File: content/en/synthetics/guide/testing-a-downloaded-file.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+  text: "Introducing Datadog Synthetics"
+- link: "synthetics/"
+  tag: "Documentation"
+  text: "Manage your checks"
+- link: "synthetics/browser_tests"
+  tag: "Documentation"
+  text: "Configure a Browser Test"
+---
+
+Journeys on web applications often involve files download: be it an order confirmation for an ecommerce website, exporting the history of a bank account’s transaction to a pdf or a csv, or downloading certificates from official authorities.
```
Comment (sarina-dd) [inline content/en/synthetics/guide/testing-a-downloaded-file.md:16]:
```suggestion
```

Moving these examples under the second paragraph and removing the intro could make this flow better.

File: content/en/synthetics/guide/testing-a-downloaded-file.md [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+
+Confirm that all these download links are working as expected using Datadog’s browser tests and the `Test a downloaded file` assertion. This means that Datadog can verify that your downloaded files are correctly being served (potentially by your FTP server), but also that they are being downloaded with the expected name, size, and that the data is not corrupted along the way.
```
Comment (sarina-dd) [inline content/en/synthetics/guide/testing-a-downloaded-file.md:18]:
```suggestion
Datadog’s browser tests and the `Test a downloaded file` assertion allow you to verify that your files are downloading correctly. For example, files downloaded from a web application, such as a PDF receipt from an e-commerce purchase or a CSV export of banking transactions, can be tested with this assertion to assure they are downloaded with the expected file name, size, and without corrupted data.
```

File: content/en/synthetics/guide/testing-a-downloaded-file.md (hunk 1 above)
Comment (sarina-dd) [inline content/en/synthetics/guide/testing-a-downloaded-file.md:16]:
```suggestion
A web application journey often involves downloading files such as an order confirmation from an e-commerce website or the PDF or CSV export history of bank account transactions.
```

File: content/en/synthetics/guide/testing-a-downloaded-file.md [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+
+In order to setup such browser test:
```
Comment (sarina-dd) [inline content/en/synthetics/guide/testing-a-downloaded-file.md:20]:
```suggestion
To setup a browser test with this assertion:
```

File: content/en/synthetics/guide/testing-a-downloaded-file.md (hunk 1 above)
Comment (l0k0ms) [reply content/en/synthetics/guide/testing-a-downloaded-file.md]: Good move 👍

File: content/en/synthetics/guide/testing-a-downloaded-file.md [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+
+Datadog’s browser tests and the `Test a downloaded file` assertion allow you to verify that downloadable files from your web application are correctly being served (for example, from your FTP server). With this assertion, downloadable files, such as a PDF receipt from an e-commerce website or a CSV history of banking transactions, can be tested to ensure they have the correct file name, size, and data.
+
+To setup a browser test with this assertion:
+
+1. **Record the step that generates the file download** in your browser test. The example below shows how to record a click on a button that triggers the download of a `.docx` file:
+
+    {{< img src="synthetics/guide/testing-a-downloaded-file/recording_step.mp4" alt="Recording steps" video="true">}}
+
+2. **Add a `Test a downloadeded file` assertion** to confirm that the file was correctly downloaded:
```
Comment (MargotLepizzera) [inline content/en/synthetics/guide/testing-a-downloaded-file.md:26]:
```suggestion
2. **Add a `Test a downloaded file` assertion** to confirm that the file was correctly downloaded:
```

**Comment by MargotLepizzera** [comment]
👍

**Pull request merged**